public/app/features/explore/state/selectors.test.ts

```diff
@@ -1,92 +0,0 @@
-import { deduplicatedRowsSelector } from './selectors';
-import { LogLevel, LogsDedupStrategy } from '@grafana/data';
-import { ExploreItemState } from 'app/types';
-
-const state: any = {
```

Comment:
Can we keep the tests and just move them to the Logs component test where this functionality was moved?

Reply:
Done in dff9b57